### PR TITLE
support incremental for validatebookmark

### PR DESCRIPTION
**Pull request by ansyral**

### CODE REVIEW

File: src/Microsoft.DocAsCode.Build.Engine/PostProcessors/HtmlPostProcessContext.cs [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        public static HtmlPostProcessContext Load(IPostProcessorHost host)
+        {
+            var stream = host?.LoadContextInfo();
+            if (stream == null || host?.IsIncremental == false)
+            {
+                var context = new HtmlPostProcessContext();
+                context.PostProcessorHost = host;
+                return context;
+            }
+            using (var sr = new StreamReader(stream))
```
Comment (hellosnow) [inline src/Microsoft.DocAsCode.Build.Engine/PostProcessors/HtmlPostProcessContext.cs]: >stream [](start = 45, length = 6)

"using" this stream to dispose?

Comment (ansyral) [inline src/Microsoft.DocAsCode.Build.Engine/PostProcessors/HtmlPostProcessContext.cs]: streamreader.dispose would dispose the stream, is it redundant?

Comment (ansyral) [inline src/Microsoft.DocAsCode.Build.Engine/PostProcessors/HtmlPostProcessContext.cs]: get it. if canIncremental is false, stream would not be closed

File: src/Microsoft.DocAsCode.Build.Engine/PostProcessors/HtmlPostProcessContext.cs [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        // bookmarks mapping from output file -> bookmarks
+        public OSPlatformSensitiveDictionary<HashSet<string>> Bookmarks { get; private set; } = new OSPlatformSensitiveDictionary<HashSet<string>>();
+
+        // file mapping from output file -> src file
+        public OSPlatformSensitiveDictionary<string> FileMapping { get; private set; } = new OSPlatformSensitiveDictionary<string>();
+
+        public static HtmlPostProcessContext Load(IPostProcessorHost host)
+        {
+            var stream = host?.LoadContextInfo();
+            using (stream)
```
Comment (vwxyzh) [inline src/Microsoft.DocAsCode.Build.Engine/PostProcessors/HtmlPostProcessContext.cs]: using (var stream = host?.LoadContextInfo())

**Comment by vwxyzh** [comment]
:shipit:

**Comment by hellosnow** [comment]
:shipit:

**Pull request merged**